__tests__/app.test.js

```diff
@@ -137,3 +137,64 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe("PATCH api/reviews/:review", () => {
+  test("returns review object with updated votes", () => {
+    const input = {
+      inc_votes: 10,
+    };
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(input)
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review.votes).toBe(15);
+        expect(review).toEqual(
+          expect.objectContaining({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+          })
+        );
+      });
+  });
+  test("status:404, responds with an error message when passed a object with wrong propreties", () => {
+    const input = {
+      voting_change: 12,
+    };
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(input)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+  test("status:404, responds with an error message when passed an object with no/wrong value type", () => {
+    const input = {
+      inc_votes: "",
+    };
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(input)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+  test("status:400, responds with an error message when passed an empty object", () => {
+    const input = {};
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(input)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+});
```

Comment:
Hey Blessing, really good tests so far! Could you also just add tests for invalid review ID and non-existent review ID? 🙂